public/data/anirudhp26.json

```diff
@@ -0,0 +1,29 @@
+
```

Comment:
There seems to be an extra empty line here.
```suggestion
```

I'm not sure this is gonna be fixed by the CI automatically.